xml/System.Threading.Tasks.Dataflow/BatchedJoinBlock`2.xml

```diff
@@ -112,6 +112,7 @@
         <summary>Initializes a new <see cref="T:System.Threading.Tasks.Dataflow.BatchedJoinBlock`2" /> with the specified configuration.</summary>
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentNullException">The <paramref name="dataflowBlockOptions" /> is <see langword="null" />.</exception>
```

Comment:
It seems that the /// comments are also missing the ArgumentException that can be thrown

Reply:
I see. What would you like me to do? Can I add a completely new line directly here?

Reply:
Yes you can @carlossanlop.